test-apps/before-all-visit/cypress/e2e/spec.cy.js

```diff
@@ -5,13 +5,11 @@ describe('coverage information', () => {
     cy.visit('/')
   })
 
-  it('calls add', () => {
+  it('calls add & sub', () => {
```

Comment:
changed to be one test due to test isolation. separate test is covered in `before-each-visit`